### ISSUE TITLE
Mixture-collapse annotation turns a positive-coupling result into an exhaustive dichotomy

### ISSUE THREAD

**Issue by microprediction** [microprediction/latentreasoning#84]
## Reviewed source

- Default-branch commit: [`2b5a20dc57f6adb7c5bf82ad1903dbea45dc4a1d`](https://github.com/microprediction/latentreasoning/commit/2b5a20dc57f6adb7c5bf82ad1903dbea45dc4a1d)
- File: [`bibliography.html`, Backour entry](https://github.com/microprediction/latentreasoning/blob/2b5a20dc57f6adb7c5bf82ad1903dbea45dc4a1d/bibliography.html#L904-L910)
- Current annotation:
  > “the softmax readout and autoregressive feedback form a system that either amplifies small differences until one component dominates or contracts mixtures until they are indistinguishable, with pretrained-model rollouts reported to sit on the amplifying side.”

Primary source: [Backour, *The Dynamics of Continuous Mixture Collapse in Language Models*, arXiv:2609.02049v1](https://arxiv.org/html/2609.02049).

I searched open and closed issues for the title, arXiv identifier, and “amplifying side”; no existing issue covers this annotation.

## The two regimes are conditional results, not an exhaustive dichotomy

For a two-token mixture, the paper derives the recurrence

```
u_(t+1) = tanh((b_t + L_t u_t)/2).
```

Its two theorems require materially narrower assumptions:

- persistent polarization: `b_t = 0` and `L_t >= L_min > 2` at every step;
- contraction: `0 <= L_t <= L_max < 2`, with exact convergence to zero only when `b_t = 0`.

Neither theorem covers negative coupling, sign-changing coupling, threshold-crossing sequences, or general nonzero fields. The site's “either … or” turns these sufficient conditions into an exhaustive classification.

### Explicit counterexample from the same recurrence

Set `b_t = 0` and constant `L_t = -4`. Then

```
u_(t+1) = tanh(-2 u_t).
```

Let `m > 0` solve `m = tanh(2m)`; numerically `m ≈ 0.9575`. The states `+m` and `-m` form a period-two orbit:

```
+m -> -m -> +m.
```

It is stable because the magnitude of the derivative at either point is

```
2(1 - m^2) ≈ 0.166 < 1.
```

This regime neither settles on one dominant component nor contracts distinct mixtures until they are indistinguishable. The dominant component alternates. Time-varying coupling and nonzero fields allow still more behavior.

The paper's measured examples may legitimately inhabit the positive, supercritical regime; that does not make the two positive-coupling theorems a general dichotomy for softmax feedback.

## “Pretrained-model rollouts” needs operational qualification

Section 4 does not propagate a continuous mixture through the nonlinear transformer at each step. To isolate recursive softmax, it:

1. runs the pretrained model on two pure branches;
2. takes each branch's greedy next token;
3. linearly interpolates the two endpoint logit vectors by construction;
4. applies softmax and updates a two-component weight.

The paper expressly says this removes nonlinear transformer processing from the mixture path. Figure 3 covers Qwen3.5-4B and Gemma-4-E4B-it on a 1,000-item controlled benchmark for 50 steps. Their measured `L_t` values are predominantly above 2 and `|b_t|` is smaller, and this surrogate rollout polarizes.

That is useful evidence for a softmax-feedback mechanism, but “pretrained-model rollouts sit on the amplifying side” can be read as a measurement of ordinary continuous-state rollouts through those models. It is instead an idealized pure-branch/logit-interpolation diagnostic. The paper's separate static injection experiments establish nonlinear mixture distortion, but they are a different experiment.

The paper's limitations also say the empirical analysis is primarily binary, covers two model families, and studies mixture preservation rather than downstream performance in a fully trained continuous-reasoning system.

## Suggested correction

For example:

> In a two-token exact-linear-logit surrogate, Backour derives `u_(t+1)=tanh((b_t+L_tu_t)/2)`. With zero field and uniformly positive coupling, `L_t>2` persistently amplifies an initial majority, while `0<=L_t<2` contracts differences. Pure-branch/logit-interpolation rollouts from Qwen3.5-4B and Gemma-4-E4B-it lie predominantly in the measured supercritical regime and polarize. These conditions are not an exhaustive classification of softmax-feedback dynamics or a test of full continuous-reasoning rollouts.

This preserves the demonstrated mechanism and empirical result without extending two sufficient-condition theorems to all possible couplings.

**Comment by microprediction** [comment]
Confirmed and applied in 88d1db6.

Verified against arXiv:2609.02049. Theorem 1 assumes `L_t >= L_min > 2`, Theorem 2 assumes `0 <= L_t <= L_max < 2`; both also take `b_t = 0`. Nothing covers negative coupling. Your counterexample checks out: with `L_t = -4`, `u_{t+1} = tanh(-2 u_t)` has `m = tanh(2m) ~ 0.9575`, and `{+m, -m}` is a stable period-2 orbit since the two-step derivative is `(2(1-m^2))^2 ~ 0.028 < 1`. A third regime, outside both.

One thing your issue does not mention, which I have accounted for: the paper's own abstract uses the either/or phrasing almost verbatim, so the site inherited it rather than inventing it. That does not make it right, and the entry now states the hypotheses instead of the rhetoric. I also added the surrogate's scope, since the rollout evidence forms the mixture in logit space rather than propagating it.
